lib/src/core/util/constants.dart

```diff
@@ -3,16 +3,8 @@ import 'package:flutter/material.dart';
 abstract class Constants {
```

Comment:
This constants file could be divided in different files, one relative to dimens (Like fontSize, height, Colors), the second one relative to String constants, another relative to api service calls (apiBaseUrl, api endpoints ..), and other only to assets.

Reply:
Done, I divided that file in Styles, Strings, ApiProperties and Assets.